spec-main/extensions-spec.ts

```diff
@@ -24,10 +25,20 @@ describe('chrome extensions', () => {
 
   afterEach(closeAllWindows)
   it('loads an extension', async () => {
-    (session.defaultSession as any).loadChromeExtension(path.join(fixtures, 'extensions', 'simple'))
-    const w = new BrowserWindow({show: false})
+    const customSession = session.fromPartition(`persist:${require('uuid').v4()}`);
+    (customSession as any).loadChromeExtension(path.join(fixtures, 'extensions', 'red-bg'))
```

Comment:
Should there be a TODO dropped in here to add type definitions for `loadChromeExtension` when the extensions build flag is toggled on?

Reply:
I expect the API surface will change substantially between now and then so I'm inclined to leave it for now.